CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -1437,6 +1441,7 @@
       </trans-unit>
       <trans-unit id="Theme" xml:space="preserve">
         <source>Theme</source>
+        <target state="translated">Anzeigemodus</target>
```

Comment:
I've found the following alternatives:

- iOS itself uses "Erscheinungsbild"
- Github uses "Darstellung"

Reply:
Changed to iOS standard. Thanks :)